web/integration-test/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
For the web module, should you also do like trino module, to have two submodules under the web module?

Reply:
What's your opinion on this?